File: apps/gate/gate/env.py
"""Loads the repo-root .env for local dev. On Fly, secrets come from the
environment directly and the file simply doesn't exist."""

from pathlib import Path

from dotenv import load_dotenv

REPO_ROOT = Path(__file__).resolve().parents[3]
load_dotenv(REPO_ROOT / ".env")

File: apps/gate/src/env.ts
// Loads the repo-root .env for local dev. On Fly, secrets come from the
// environment directly and the file simply doesn't exist.
import { fileURLToPath } from "node:url";
import dotenv from "dotenv";

dotenv.config({ path: fileURLToPath(new URL("../../../.env", import.meta.url)), quiet: true });
